lib/ansible/plugins/connection/psrp.py

```diff
@@ -94,6 +94,13 @@
     vars:
     - name: ansible_psrp_connection_timeout
     default: 30
+  read_timeout:
+    description:
+    - The read timeout for receiving data from the remote host.
+    - This is measured in seconds.
```

Comment:
Good to add another line saying `This value should always be greater than I(operation_timeout)` and that it requires pypsrp<=0.3.0.`